.github/workflows/e2e-tests-wp-nightly.yml

```diff
@@ -5,7 +5,61 @@ on:
     branches:
       - develop
       - main
+    # Ignore Storybook-only and unit-test-only changes when checking to run
+    # E2E tests. This is useful when adding/modifying/refactoring/removing
+    # test code ONLY in a commit, but saves running the expensive E2E test
+    # suite on GitHub Actions when all that's changing are tests and not code
+    # that would affect E2E behavior.
+    paths:
+      - '.github/workflows/e2e-tests-wp-nightly.yml'
+      - '.babelrc'
+      - 'assets/**'
+      - 'includes/**'
+      - 'google-site-kit.php'
+      - 'jest-puppeteer.config.js'
+      - 'composer.json'
+      - 'composer.lock'
+      - 'package.json'
+      - 'package-lock.json'
+      - 'postcss.config.js'
+      - 'scoper.inc.php'
+      - 'uninstall.php'
+      - 'webpack.config.js'
+      - 'tests/e2e/**'
+      - '!assets/**/__tests__/**/*.js'
+      - '!assets/**/test/*.js'
+      - '!assets/**/*.test.js'
+      - '!packages/**/__tests__/**/*.js'
+      - '!packages/**/test/*.js'
+      - '!packages/**/*.test.js'
   pull_request:
+    # Ignore Storybook-only and unit-test-only changes when checking to run
+    # E2E tests. This is useful when adding/modifying/refactoring/removing
+    # test code ONLY in a commit, but saves running the expensive E2E test
+    # suite on GitHub Actions when all that's changing are tests and not code
+    # that would affect E2E behavior.
+    paths:
+      - '.github/workflows/e2e-tests-wp-nightly.yml'
+      - '.babelrc'
+      - 'assets/**'
+      - 'includes/**'
+      - 'google-site-kit.php'
+      - 'jest-puppeteer.config.js'
+      - 'composer.json'
+      - 'composer.lock'
+      - 'package.json'
+      - 'package-lock.json'
+      - 'postcss.config.js'
+      - 'scoper.inc.php'
```

Comment:
We should also include `/php-scoper/composer.lock` since we manage the installation of php-scoper separately here.